### DISCUSSION TITLE
scrolling is not working in modal mode with detox

### DISCUSSION THREAD

**Discussion by SteveTro** [henninghall/react-native-date-picker#648] [General]
I try to use detox to test a screen which contains an react-native-date-picker as modal.
The problem is that no testID, accessibilyLabel or something is provided in the modal version.

I tried it with the inline picker and was able to swipe the picker but i can not get the picker in the modal. 
I tried to analyze the structure with xCode hierarchical debug, but doesn't matter if i try to select by type "DatePicker", _UIDatePickerView or i even tried to select by "UIPickerColumnView", i can not make the scroll work.

In the repo there is just a example how to test if the picker is visible, not to change the date.

Is there a way how to archive this?

```javascript
export default class App extends Component {

<DatePicker
        modal
        maximumDate={maximumDate}
        minimumDate={minimumDate}
        open={open}
        testID={`${testID}.datePicker`}
        date={new Date(value ?? moment().format())}
        locale="en-US"
        mode={mode}
        theme={isDarkMode ? 'light' : 'light'}
        onConfirm={(date) => {
          setOpen(false);
          onConfirm(date.toUTCString());
        }}
        onCancel={() => setOpen(false)}
        {...props}
      />

}
```

 - OS: iOS
 - React Native version 0.68
 - react-native-date-picker version 4.2.2

**Comment by Bucci83** [comment]
hello, did you ever found how to do it? Im stuck with the same problem, I cannot perform a real E2E testing simulating the user interaction.
iOS supports ".setDatePickerDate" https://wix.github.io/Detox/docs/19.x/api/actions-on-element#setdatepickerdatedatestring-dateformat--ios-only
but with Android there is no way ???
